tests/e2e/hcp_external_auth_test.go

```diff
@@ -101,26 +108,13 @@ var _ = Describe("Rosacli Testing", func() {
 	})
 
 	Describe("Hypershift external auth validation testing", func() {
-		BeforeEach(func() {
-			Expect(clusterID).ToNot(BeEmpty(), "Cluster ID is empty, please export the env variable CLUSTER_ID")
-			rosaClient = rosacli.NewClient()
-			clusterService = rosaClient.Cluster
-		})
-
-		AfterEach(func() {
-			By("Clean remaining resources")
-			err := rosaClient.CleanResources(clusterID)
-			Expect(err).ToNot(HaveOccurred())
-		})
 
 		It("validation for HCP cluster break_glass_credentials create/list/describe/delete can work well via ROSA client - [id:73018]",
 			labels.Medium,
 			func() {
 				By("Create/list/revoke break-glass-credential to non-HCP cluster")
-				hosted, err := clusterService.IsHostedCPCluster(clusterID)
-				Expect(err).ToNot(HaveOccurred())
 
-				if !hosted {
+				if !hostedCluster {
```

Comment:
We have an issue in this case. We are in HCP testing file and we test classic ... There is a mix which should not happen IMO